evalml/data_checks/default_data_checks.py

```diff
@@ -32,7 +32,7 @@ def __init__(self, problem_type):
         Arguments:
             problem_type (str): The problem type that is being validated. Can be regression, binary, or multiclass.
         """
-        if handle_problem_types(problem_type) == ProblemTypes.REGRESSION:
+        if handle_problem_types(problem_type) in [ProblemTypes.REGRESSION, ProblemTypes.TIME_SERIES_REGRESSION]:
```

Comment:
This should have been done in #1378 😬

Reply:
Good catch!

We must've been missing unit test coverage for the default data checks for the time series regression problem type. Could we add that? Should just be able to clone an existing test, ensure the right data checks show up just like regression.

Reply:
I see you added that. Champion! 🏅  🤣